lib/rules/no-confusing-arrow.js

```diff
@@ -54,6 +55,7 @@ module.exports = {
     create(context) {
         const config = context.options[0] || {};
         const allowParens = config.allowParens || (config.allowParens === void 0);
+        const onlyOneSimpleParam = context.options[1];
```

Comment:
```suggestion
        const onlyOneSimpleParam = !!config.onlyOneSimpleParam;
```

`context.options[1]` is always undefined, we need `contex.options[0].onlyOneSimpleParam` here.

Reply:
Sure! Changing it :)